root/static/scripts/tests/Control/URLCleanup.js

```diff
@@ -4121,6 +4121,12 @@ limited_link_type_combinations: [
     expected_relationship_type: 'soundcloud',
             expected_clean_url: 'https://soundcloud.com/alisonwonderland',
   },
+  {
+                     input_url: 'https://soundcloud.com/erin-thomson-776648435?utm_source=clipboard&utm_medium=text&utm_campaign=social_sharing',
+             input_entity_type: 'artist',
+    expected_relationship_type: 'soundcloud',
+            expected_clean_url: 'https://soundcloud.com/erin-thomson-776648435',
+  },
```

Comment:
The earlier test with `metro-luminal` might be no longer needed as it’s quite redundant with this one.